lib/Hooks/UserHooks.php

```diff
@@ -286,6 +286,10 @@ public function setPassphrase($params) {
 
 				$keyPair = $this->crypt->createKeyPair("oc:uid:$user");
 
+				if ($keyPair === null) {
+					return null;
```

Comment:
Probably need to log something here (like in line 306)
And understand the reason that `createKeyPair` returns `null` in some situations.